README.md

```diff
@@ -1,93 +1,284 @@
+
+## Project Teams Details
 
 A Project Team is a sub-team with a dynamic charter, except for a few caveats as detailed here. They exist to maintain a specific project (or possibly set of projects). Each project in `conda` or `conda-incubator` should belong to exactly one Project Team. This team is granted write access to the repos associated with the project.
 
-The initial Project Team for projects in the `conda` organization are specified in the vote to move a project from the incubator to the main `conda` organization. As this vote has a 60% threshold and sub-team formation is only a greater than 50% threshold, the vote for adding a project to the main `conda` organization can serve both purposes. Project Teams for incubating projects do not need a vote for their creation. Instead they are created when the project is moved into `conda-incubator`.
+The initial Project Team for projects in the `conda` GitHub organization are specified in the vote to move a project from the incubator to the main `conda` GitHub organization. As this vote has a 60% threshold and sub-team formation is only a greater than 50% threshold, the vote for adding a project to the main `conda` GitHub organization can serve both purposes. Project Teams for incubating projects do not need a vote for their creation. Instead they are created when the project is moved into `conda-incubator`.
+
+For [federated](#federated-projects) and [incubating](#incubation) projects, the Project Team decides among themselves on how to add and/or remove members from the Project Team. Further, Project Teams for incubating projects are exempted from the requirement to have at least one Steering Council member. For federated projects, the Project Team must have at least one Steering Council member. For a community maintained project, a petition for commit rights is made to the current members of the Project Team for that project. If this petition passes according to the voting rules for "[Nominate new member of a Project Team](#nominate-new-member-of-a-community-project-team) below, the petitioner is granted write access and added to the team.
+
+## Steering Council Membership
+
+Steering Council members need to 1) be invested in the conda ecosystem and community, and 2) have time and energy to contribute to conda Organization governance.  Council members, with the exception of [provisional members](#provisional-members), will have a demonstrated record of prior service to the conda community.
+
+### Steering Council Size
+
+The Steering Council should reflect a wide range of interests, and still be small enough to have a reasonable expectation that [quorum requirements](#quorum) can be met when votes are called.
+
+The minimum and maximum sizes for the Steering Council are:
+
+* Minimum: 9
+* Maximum: 21
```

Comment:
I would add that if we have an upper limit, it would make sense to limit the number of participants from companies that are being paid to work for the company interest? so the council will not get controlled by any specific company and interests of them?

Reply:
@xhochy I added the upper limit after speaking with the IP lawyer Van Lindberg.  He argued that having an upper limit would encourage commercial participation (which is one of the goals of the update). If we don't have an upper limit then commercial entities will have no idea how dilute their influence can become. I hesitate to disagree with Van on anything...

We could bump this number up though, and still address the concern that Van raised.

Reply:
@marcelotrevisani Right now this proposal has hard limits 

* on the number of council members from the same company (2).  
* on the total number of council members that can be provisional members (33%).

Non-provisional members are members because they have a demonstrated history of contributing to the conda ecosystem.  You would never have a provisional member and a non-provisional member from the same company.

Does that address your concerns?

Reply:
> @marcelotrevisani Right now this proposal has hard limits 
> 
> 
> 
> * on the number of council members from the same company (2).  
> 
> * on the total number of council members that can be provisional members (33%).
> 
> 
> 
> Non-provisional members are members because they have a demonstrated history of contributing to the conda ecosystem.  You would never have a provisional member and a non-provisional member from the same company.
> 
> 
> 
> Does that address your concerns?  

Yes, it does. Thank you! :)

Reply:
I agree @tnabtaf. We already tolerate that risk and that seems fine.

Reply:
@xhochy You ok here or do you have more questions?

Reply:
@xhochy: We dropped the maximum number of board members.  This helped us address several of the concerns that were raised by @CJ-Wright below.

Reply:
Thanks, this makes me feel much better about the proposal!